index.html

```diff
@@ -988,6 +988,10 @@
         </p>
         <p>Special thanks to those that helped pioneer the first edition of this model, including especially: Charu Chandiram, Peter Freiling, Nathan Furtwangler, Thomas Olsen, Matt Rakow, Ramu Ramanathan, Justin Rogers, Jacob Rossi, Reed Townsend and Steve Wright.</p>
     </section>
+    <section class="appendix">
+        <h1>Security and Privacy Considerations</h1>
+        <p>There are no known security or privacy impacts resulting from the features outlined in this specification.</p>
```

Comment:
I was expecting:

The working group has undertaken the Privacy and Security questionnaire [[!CITATION HERE]]. From the review, the working group believes that this API: 

 * Does not  expose personally-identifiable information.
 * Does not deal with high-value data.
 * Does not introduce new state for an origin that persists across browsing sessions.
 * Does not expose persistent, cross-origin state to the web. 
 * Does not expose any other data to an origin that it doesn’t currently have access to.
 * Does not enable new script execution/loading mechanisms.
 * Does not allow an origin access to a user’s location.
 * Does not require any special handling when the user agent is in "incognito" mode.
* Does not allow an origin access to other devices.
* Does not allow an origin control over a user agent’s native UI.
* Does not expose temporary identifiers to the web.
* Does not distinguish between behavior in first-party and third-party contexts.
* Does not persist data to a user’s local device.
* Does not allow downgrading default security characteristics.

However, ANSWER THESE please: 

* Does not allow an origin access to sensors on a user’s device?
* Does not allow an origin access to aspects of a user’s local computing environment?